AdyenDropIn/Views/ModalToolbar.swift

```diff
@@ -91,21 +91,21 @@ internal final class ModalToolbar: UIView {
     }
 
     private func setupLayoutForCenteredMode() {
```

Comment:
Does this work for all the cases?
Does `rightCancel` still make sense for Arabic?

Reply:
it works yes, and for arabic speakers cancel on the left make sense yes, like when UINavigationController put the back button on the left and viewController's are push from right to left.

Reply:
that is what I mean:
If it says "left", but it is actually on the right - it could be confusing 😅

We can probably enforce `rightCancel` or `leftCancel` to be actually on left on to the right respectily.. or rename rightCancel` or `leftCancel` to something else ?

Reply:
fixed by adding `ToolbarMode.natural` and made it the default.